demisto_sdk/commands/lint/resources/pylint_plugins/base_checker.py

```diff
@@ -35,6 +41,7 @@ class CustomBaseChecker(BaseChecker):
 
     def __init__(self, linter=None):
         super(CustomBaseChecker, self).__init__(linter)
+        self.commands = os.getenv('commands').split(',') if os.getenv('commands') else []
```

Comment:
please add an explanation why it is from the os.getenv and you can not pass it from the `_facts`

Reply:
added in the linter file, when setting the env vars.